common/services/event.js

```diff
@@ -0,0 +1,85 @@
+const restClient = require('../lib/api-client/rest-client')
+
+const BaseService = require('./base')
+class EventService extends BaseService {
+  postEvents(lockoutEvents, move, user) {
+    const PER_EVENTS = [
+      'PerViolentDangerous',
+      'PerWeapons',
+      'PerConcealed',
+      'PerSelfHarm',
+      'PerEscape',
+      'PerMedicalAid',
+      'PerMedicalDrugsAlcohol',
+      'PerMedicalMentalHealth',
+      'PerPropertyChange',
+      'PerGeneric',
+    ]
+    const PERSON_MOVE_EVENTS = [
+      'PersonMoveUsedForce',
+      'PersonMoveDeathInCustody',
+    ]
+
+    const todaysDate = new Date()
+    const events = []
+
+    events.push(lockoutEvents.events)
+
+    events.flat().map(async event => {
+      let type
+      let relationshipsId
+      let supplierPersonnelNumber
+
+      const eventDescription = lockoutEvents[event]
+
+      if (PER_EVENTS.includes(event)) {
+        type = 'person_escort_records'
+        relationshipsId = move.profile.person_escort_record.id
+        supplierPersonnelNumber = 'supplier_personnel_number'
```

Comment:
```suggestion
        supplierPersonnelNumber = 'police_personnel_number'
```

As we're sending this up as police, it should be the police personnel number.